examples/github-minimal.yml

```
@@ -0,0 +1,28 @@
+name: Build
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    name: Build wheel on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04, windows-latest, macos-latest]
+    steps:
+    - uses: actions/checkout@v1
+    - uses: actions/setup-python@v1
+      name: Install Python ${{ matrix.python_version }}
+      with:
+        python-version: "3.7"
+
+    - name: Install cibuildwheel
+      run: |
+        python -m pip install cibuildwheel==1.2.0
+    - name: Install Visual C++ for Python 2.7
+      if: startsWith(matrix.os, 'windows')
+      run: |
+        choco install vcpython27 -f -y
+    - name: Build wheel
+      run: |
+        python -m cibuildwheel --output-dir wheelhouse
```

It would be nice to keep the wheels as artifacts in the minimal config, I think

```suggestion
        python -m cibuildwheel --output-dir wheelhouse
    - uses: actions/upload-artifact@v1
       with:
         name: wheels
         path: ./wheelhouse
```

---

Definitely keep the wheels as artifacts, especially as uploading wheels to PyPI using `pypa/gh-action-pypi-publish` only works on Linux, so you have to:

- upload wheels for the various platforms as artifacts
- download all the wheels in a job running on Linux
- upload them to PyPI

NOTE: I'm not sure what happens if you give the same name to uploads from all platforms

---

>I'm not sure what happens if you give the same name to artifacts from all platforms

It's not a problem as long as the file names are unique.